joomla/joomla-cms
# Use the new useglobal form attribute in captcha config options fields

Pull Request for Improvement.

### Summary of Changes

This PR makes use of the new use global on the capatcha fields in com_users, com_contact and com_content captcha config options.

### Testing Instructions

1. Use latest staging and apply patch
2. Confirm with/without captacha plugins enabled the options still work and properly inherit the setting from global config.
3. Confirm capctha is, when enabled, still displayed in all frontend scenarios (create new article, submit contact form, users register, user remimd username, user password reset)

#### Before
![image](https://user-images.githubusercontent.com/9630530/26888435-ed2a9ea4-4ba2-11e7-851c-bdb4cdae4b00.png)

#### After
![image](https://user-images.githubusercontent.com/9630530/26888402-d49efe2a-4ba2-11e7-9356-db880ba6a1a9.png)

### Documentation Changes Required

None.
